tests/conftest.py

```diff
@@ -349,6 +350,13 @@ def application_docker_image(
     runtime: str,
     application_image_tag: str,
 ) -> Iterable[Image]:
+    if platform.machine() == "aarch64":
+        if application_image_tag == "j9" or application_image_tag == "zing":
+            pytest.xfail(
+                "Different JVMs are not supported on aarch64, see https://github.com/Granulate/gprofiler/issues/717"
+            )
+        if application_image_tag == "musl":
+            pytest.xfail("musl is not supported on aarch64, see https://github.com/Granulate/gprofiler/issues/717")
```

Comment:
Is this the right link? https://github.com/Granulate/gprofiler/issues/717 doesn't mention anything about musl.